assets/header_template.go

```diff
@@ -0,0 +1,24 @@
+package assets
+
+var CLANG_HTTP_POST_HEADER = 
```

Comment:
golang has a very handy string interpolation tool.  the back quote.

Try this example I created:  https://play.golang.org/p/Fj5rBk4XBB